code/__defines/subsystems.dm

```diff
@@ -20,27 +20,28 @@
 // Subsystems shutdown in the reverse of the order they initialize in
 // The numbers just define the ordering, they are meaningless otherwise.
 
-#define SS_INIT_INPUT            22
-#define SS_INIT_EARLY            21
-#define SS_INIT_WEBHOOKS         20
-#define SS_INIT_MODPACKS         19
-#define SS_INIT_SECRETS          18
-#define SS_INIT_GARBAGE          17
-#define SS_INIT_MATERIALS        16
-#define SS_INIT_PLANTS           15
-#define SS_INIT_LORE             14
-#define SS_INIT_MISC             13
-#define SS_INIT_SKYBOX           12
-#define SS_INIT_MAPPING          11
-#define SS_INIT_JOBS             10
-#define SS_INIT_CIRCUIT          9
-#define SS_INIT_GRAPH            8
-#define SS_INIT_OPEN_SPACE       7
-#define SS_INIT_ATOMS            6
-#define SS_INIT_PRE_CHAR_SETUP   5
-#define SS_INIT_CHAR_SETUP       4
-#define SS_INIT_MACHINES         3
-#define SS_INIT_ICON_UPDATE      2
+#define SS_INIT_INPUT            23
+#define SS_INIT_EARLY            22
+#define SS_INIT_WEBHOOKS         21
+#define SS_INIT_MODPACKS         20
+#define SS_INIT_SECRETS          19
+#define SS_INIT_GARBAGE          18
+#define SS_INIT_MATERIALS        17
+#define SS_INIT_PLANTS           16
+#define SS_INIT_LORE             15
+#define SS_INIT_MISC             14
+#define SS_INIT_SKYBOX           13
+#define SS_INIT_MAPPING          12
+#define SS_INIT_JOBS             11
+#define SS_INIT_CIRCUIT          10
+#define SS_INIT_GRAPH            9
+#define SS_INIT_OPEN_SPACE       8
+#define SS_INIT_ATOMS            7
+#define SS_INIT_PRE_CHAR_SETUP   6
+#define SS_INIT_CHAR_SETUP       5
+#define SS_INIT_MACHINES         4
+#define SS_INIT_ICON_UPDATE      3
+#define SS_INIT_VIS_CONTENTS     2
```

Comment:
It should init after `SS_INIT_AIR` to avoid wasted updates before zones are built.